lib/api/api.go

```diff
@@ -1253,27 +1137,27 @@ func (s *apiService) postDBIgnores(w http.ResponseWriter, r *http.Request) {
 	s.getDBIgnores(w, r)
 }
 
-func (s *apiService) getIndexEvents(w http.ResponseWriter, r *http.Request) {
-	s.fss.gotEventRequest()
+func (s *service) getIndexEvents(w http.ResponseWriter, r *http.Request) {
+	s.fss.GotEventRequest()
```

Comment:
Again, unrelated, feel free to ignore, but this feels like a callback hence should be "On<Something>" rather than just "Something()"

Reply:
Get the callback bit, but not the "On" vs "Something()" part.

Reply:
Actually, this should probably just be called Invalidate()

Reply:
That I understand even less. The summary service records the time that `GotEventRequest` was last called as the time when an event was last requested - how does that invalidate anything?

Reply:
Yeah, you are right.
GotEventRequest() is like saying "what I got" rather than saying I call your "OnEventRequest()" when I get something. Essentially what I am saying is that GotEventRequest() feels like an accessor to some boolean flag.